Client/Client.Qt/Widgets/TitleWidget.cpp

```diff
@@ -22,20 +22,20 @@ TitleWidget::TitleWidget(QWidget* parent, const Style::TitleBar& st) : QWidget(p
 
     connect(window()->windowHandle(), &QWindow::windowStateChanged, this, [=](Qt::WindowState state) { windowStateChanged(state); });
 
-    _maximizeButton->setClickCallback([=]() {
+    _maximizeButton->setClickCallback([parent, this]() {
         if (parent->isMaximized())
-            window()->setWindowState(Qt::WindowNoState);
+            this->window()->setWindowState(Qt::WindowNoState);
         else
-            window()->setWindowState(Qt::WindowMaximized);
-        _maximizeButton->clearState();
+            this->window()->setWindowState(Qt::WindowMaximized);
+        this->_maximizeButton->clearState();
     });
-    _closeButton->setClickCallback([=]() {
+    _closeButton->setClickCallback([parent, this]() {
         parent->deleteLater();
-        _closeButton->clearState();
+        this->_closeButton->clearState();
     });
-    _minimizeButton->setClickCallback([=]() {
+    _minimizeButton->setClickCallback([parent, this]() {
         parent->showMinimized();
-        _minimizeButton->clearState();
+        this->_minimizeButton->clearState();
```

Comment:
This inside lambda body overloads code

Reply:
now, it looks good, we specify what objects we work with and how we transfer them, depending on the task